core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.19.0.groovy

```diff
@@ -24,4 +24,70 @@ schemaChange {
   cql """
 ALTER TABLE data WITH COMPRESSION = {'sstable_compression': 'DeflateCompressor'};
 """
-}
\ No newline at end of file
+}
+
+schemaChange {
+  version '3.1'
+  author 'jsanda'
+  tags '0.19.x'
+  cql """
+CREATE TABLE rollup60 (
+    tenant_id text,
+    metric text,
+    shard  bigint,
+    time timestamp,
+    min double,
```

Comment:
Instead of min, max, avg, median (which is 50% percentile, so it should be with percentiles..), sum, percentileA, percentileB etc.. soon someone will want "std deviation", "mode", "variance" and so on. And we always need new columns and we might only want to calculate part of the information as well. I would like to avoid that, instead getting a map of different aggregates (like the data table placeholder) which would then be returned to the requester.

Same applies to the more tables, user configurable rollup periods is something we could relatively easily provide and they would always use the same table -> no change in schema required if everything is in same table.